src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../App';
+
+import usersFromServer from '../../api/users';
```

Comment:
Avoid importing `usersFromServer` directly in this component. Instead, pass the necessary user data as props from the parent component to ensure better data flow and maintainability.